quality-assurance/performance-test-checklist.md

```diff
@@ -0,0 +1,80 @@
+# Performance testing checklist
+
+## Purpose of this document
+This document lists activities that you should consider when planning performance testing
+
+## Activity list
+
+### Establish business requirements
+- What are the requirements for resilience under load (e.g. availability, responsiveness)
+- What will be typical and peak usage of the system
+- What are the requirements for scaling
+- What are typical patterns of usage (e.g. peaks, types of request)
+
+### Agree types of performance testing to carry out, and data to gather
+e.g.
+- Load testing
+- Scalability testing
+- Soak testing
+- Stress Testing
+etc.
```

Comment:
Argh you used etc. My eyes.

Reply:
I've removed.  One day you must explain to me your beef with etc.